core/templates/pages/community-dashboard-page/modal-templates/question-suggestion-editor-modal.directive.html

```diff
@@ -5,6 +5,15 @@ <h3><[skill.getDescription()]></h3>
 
   <div class="modal-body">
     <div class="oppia-question-editor-section">
+      <h3>Selected Rubric: <[skillDifficultyString]></h3>
+      <div ng-if="skill.getRubricExplanations(skillDifficultyString).length > 0">
+        <h3>Explanations</h3>
```

Comment:
Maybe 'Difficulty Explanations'? Ditto below. If I was a new user, I would be confused what 'explanations' is referring to.